### PR TITLE
Add print selected LaboratorySamples in list

### PR DESCRIPTION
**Pull request by ftarulla**
We add checkboxes to the LaboratorySamples list so that the user may select samples. Then the selected samples may be printed in a bulk action.

### CODE REVIEW

File: app/models/concerns/multipage_pdf_renderer.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+require 'open3'
```
Comment (matiasgarciaisaia) [inline app/models/concerns/multipage_pdf_renderer.rb:1]: Oh my.

File: app/models/concerns/multipage_pdf_renderer.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      file.write(doc)
+      file.rewind
+      file
+    end
+
+    filepaths = tempfiles.map{ |tf| tf.path.to_s }
+
+    wickedPdf = WickedPdf.new
+    binary = wickedPdf.send(:find_wkhtmltopdf_binary_path)
+    options = wickedPdf.send(:parse_options, options)
```
Comment (matiasgarciaisaia) [inline app/models/concerns/multipage_pdf_renderer.rb:20]: Is this Ruby's `send` to send the `find_wkhtmltopdf_binary_path` message? Is there any reason to not directly invoke it (ie, `wickedPdf.find_wkhtmltopdf_binary_path`)?

File: app/models/concerns/multipage_pdf_renderer.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    command = [binary, '-q']
+    command += options
+    filepaths.each { |fp| command << fp }
+    command << outfile.path.to_s
+    err = Open3.popen3(*command) do |stdin, stdout, stderr|
+      stderr.read
+    end
+
+    raise "Problem generating multipage pdf: #{err}" if err.present?
```
Comment (matiasgarciaisaia) [inline app/models/concerns/multipage_pdf_renderer.rb:30]: Are we handling this exception somewhere?

**Review by matiasgarciaisaia — APPROVED** [review]
Great one!

File: app/models/concerns/multipage_pdf_renderer.rb (hunk 2 above)
Comment (ftarulla) [reply app/models/concerns/multipage_pdf_renderer.rb:20]: It's necessary to invoke private messages.

**Review by matiasgarciaisaia — APPROVED** [review]
🚀 🍟

**Pull request merged**